docs/src/folderstruct.md

```diff
@@ -0,0 +1,55 @@
+# ClimaLSM folder structure
+
+This documentation page is meant to help new users navigate through the repository of [ClimaLSM](https://github.com/CliMA/ClimaLSM.jl).
+
+`ClimaLSM` home directory has 5 main folders:<br>
+- docs: contains files to generate [the documentation website](clima.github.io/ClimaLSM.jl/) you are reading right now.  
+- experiments: contains simple runs of `ClimaLSM` models. 
+- parameters: contains a file to retrieve constants such as avogadro's number, the speed of light, etc. 
+- src: contains the code of `ClimaLSM` models. 
+- test: contains [unit tests](https://en.wikipedia.org/wiki/Unit_testing), which are meant to ensure small pieces of ClimaLSM source code works as intended before merging pull requests.
+
+and 3 GitHub actions folders. GitHub actions are `.yml` files, which are bash scripts that runs on a remote computer on each git push. 
+- .builkite: contains a script building outputs such as figures from experiments and tests folders. 
+- .dev: contains
```

Comment:
i think right now this just has the julia formatter? Not totally sure how to describe the purpose of this folder.